### PR TITLE
Greek 

### PR DESCRIPTION
**Pull request by avdelmaria**

### CODE REVIEW

**Comment by avdelmaria** [comment]
All Greek translations will be here. Ready until now
1. Digital Gold.gr.md
2. Skycoin Bounty Program 1.gr.md

File: content/statement/Digital Gold.gr.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
++++ title = "Digital Gold" tags = [ "Statement", ] date = "2017-08-31" categories = [ "Statement", ] +++
```
Comment (gz-c) [inline content/statement/Digital Gold.gr.md]: need to preserve original formatting of this section

**Comment by gz-c** [comment]
Checked by "Kostas P" in telegram

**Comment by gz-c** [comment]
15 sky

**Pull request merged**